src/app-settings/service-api.ts

```diff
@@ -51,8 +46,8 @@ const plugin: FastifyPluginAsync<PluginOptions> = async (fastify, options) => {
       prefix: '/app-settings',
       shouldRedirectOnDownload: false,
       uploadMaxFileNb: 1,
-      serviceMethod: serviceMethod,
-      serviceOptions: {
+      fileItemType: fileItemType,
```

Comment:
Same-name shorthand?
```suggestion
      fileItemType,
```